code/modules/reagents/chemistry/reagents/cat2_medicine_reagents.dm

```diff
@@ -6,32 +6,34 @@
 /******BRUTE******/
 /*Suffix: -bital*/
 
-/datum/reagent/medicine/C2/helbital //only REALLY a C2 if you heal the other damages but not being able to outright heal the other guys is close enough to damaging
+/datum/reagent/medicine/C2/helbital //kinda a C2 only if you're not in hardcrit.
 	name = "Helbital"
-	description = "Named after the norse goddess Hel, this medicine heals the patient's bruises the closer they are to death. Burns, toxins, and asphyxition will increase healing but these damages must be maintained while the drug is being metabolized or the drug will react negatively."
+	description = "Named after the norse goddess Hel, this medicine heals the patient's bruises the closer they are to death. Patients will find the medicine 'aids' their healing if not near death by causing asphyxiation."
 	color = "#9400D3"
 	taste_description = "cold and lifeless"
 	overdose_threshold = 35
 	reagent_state = SOLID
 	var/helbent = FALSE
-	var/beginning_combo = 0
 	var/reaping = FALSE
 
-/datum/reagent/medicine/C2/helbital/on_mob_metabolize(mob/living/carbon/M)
-	beginning_combo = M.getToxLoss() + M.getOxyLoss() + M.getFireLoss() //This DOES mean you can cure Tox/Oxy and then do burn to maintain the brute healing that way.
-	return ..()
-
 /datum/reagent/medicine/C2/helbital/on_mob_life(mob/living/carbon/M)
 	. = TRUE
-	var/cccombo = M.getToxLoss() + M.getOxyLoss() + M.getFireLoss()
-	var/healed_this_iteration = FALSE
-	if(cccombo >= beginning_combo)
-		M.adjustBruteLoss(FLOOR(cccombo/-15,0.1)) //every 15 damage adds 1 per tick
-		healed_this_iteration = TRUE
-	else
-		M.adjustToxLoss((beginning_combo-cccombo)*0.1) //If you are just healing instead of converting the damage we'll KINDLY do it for you AND make it the most difficult!
-
-	if(healed_this_iteration && !reaping && prob(0.0001)) //janken with the grim reaper!
+	var/death_is_coming = (M.getToxLoss() + M.getOxyLoss() + M.getFireLoss() + M.getBruteLoss())
+	var/thou_shall_heal = 0
+	var/good_kind_of_healing = FALSE
+	switch(M.stat)
+		if(CONSCIOUS) //bad
+			thou_shall_heal = death_is_coming/50
+			M.adjustOxyLoss(2, TRUE)
+		if(SOFT_CRIT) //meh convert
+			thou_shall_heal = round(death_is_coming/47,0.1)
+			M.adjustOxyLoss(1, TRUE)
+		else //no convert
+			thou_shall_heal = round(death_is_coming/45,0.1)
+			good_kind_of_healing = TRUE
+	M.adjustBruteLoss(thou_shall_heal, FALSE)
+
+	if(good_kind_of_healing && !reaping && prob(0.0001)) //janken with the grim reaper!
```

Comment:
I still think that you should up this prob(0.0001) to something like, I dunno, prob(0.1) (or even prob(1)) or something, because nobody is ever, EVER gonna see this feature otherwise, especially since helbital isn't in Medbay at roundstart and has to be specifically made by a chemist for anyone to even have a CHANCE to get to do the grim reaper bet thing (and then they have to be in hardcrit and roll a chance (which they can only roll once every 2 seconds) that's about as rare as winning a pulse rifle from an arcade machine, then actually accept the bet, even though they're probably being healed by a doctor if they still have helbital in their system while in hardcrit).

Reply:
no